logger.gd

```diff
@@ -6,6 +6,58 @@
 
 extends Node # Needed to work as a singleton
 
+const error_messages = {
```

Comment:
This is fine for the purpose of this plugin, but in the long run I think it might be worth having this kind of stringification of error codes directly in the engine itself, as many users would benefit from an easy translation of error numbers to human readable strings.